src/styles/styles.js

```diff
@@ -1260,7 +1260,7 @@ const styles = {
     },
 
     fullScreenLoading: {
-        backgroundColor: themeColors.modalBackdrop,
+        backgroundColor: themeColors.componentBG,
         opacity: 0.8,
```

Comment:
Hmm actually I think maybe we can just use a solid background or just get rid of the effect entirely on wider screens. There is still a weird flashing effect I think...

**With transparency**
https://user-images.githubusercontent.com/32969087/114217650-24b9b000-9904-11eb-97c4-85b6c9758c1a.mp4

**Without transparency**
https://user-images.githubusercontent.com/32969087/114217700-3307cc00-9904-11eb-876f-ba573622502f.mp4

**No loader**
https://user-images.githubusercontent.com/32969087/114217721-3b600700-9904-11eb-8dd2-0b1c280fe0e4.mp4


cc @shawnborton

Reply:
Resolving this as per https://github.com/Expensify/Expensify.cash/pull/2322#issuecomment-819049488